### PR TITLE
doc: use "*_by_lua_block" in sample codes instead of "*_by_lua"

### PR DESCRIPTION
**Pull request by philipjkim**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

**Comment by agentzh** [comment]
@philipjkim Thanks for the patch but I'm afraid you are editing the wrong file. To quote the first few lines at the top of the content of `README.markdown`:

```
<!---
Don't edit this file manually! Instead you should generate it by using:
    wiki2markdown.pl doc/HttpLuaModule.wiki
-->
```

File: README.markdown [hunk 1]
```diff
@@ -1268,33 +1296,6 @@ Basically you can safely use Lua libraries that do blocking I/O in this very con
 
 You should be very careful about potential security vulnerabilities in your Lua code registered in this context because the Nginx master process is often run under the `root` account.
 
-This directive was first introduced in the `v0.5.5` release.
-
-[Back to TOC](#directives)
-
-init_by_lua_block
```
Comment (agentzh) [inline README.markdown]: Why removing this section?

**Comment by agentzh** [comment]
@philipjkim I prefer not moving the content from `*_by_lua` to `*_by_lua_block` in this PR since it makes the diff much much harder to review. Too many changes.

**Comment by philipjkim** [comment]
@agentzh Please ignore this PR. I'll re-request, applying your comments.

**Pull request closed without merging**